evap/results/tests/test_views.py

```diff
@@ -52,6 +52,22 @@ def test_multiple_evaluations_per_course(self):
         self.assertNotContains(page, evaluation2.full_name)
         caches['results'].clear()
 
+
+    @patch('evap.evaluation.models.Evaluation.can_be_seen_by', new=(lambda self, user: True))
+    def test_order(self):
+        student = baker.make(UserProfile, email="student@institution.example.com")
+
+        course = baker.make(Course)
+        evaluation1 = baker.make(Evaluation, name_de='random_evaluation_d', name_en='random_evaluation_a', course=course, state='published')
+        evaluation2 = baker.make(Evaluation, name_de='random_evaluation_c', name_en='random_evaluation_b', course=course, state='published')
+
+        page = self.app.get(self.url, user=student).body.decode()
+        self.assertTrue(page.index(evaluation1.name_en) < page.index(evaluation2.name_en))
```

Comment:
We should also be able to use `self.assertLess` and `self.assertGreater` here (see [docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLess)), which would be a little more precise I think.